### PR TITLE
 Make openstack rootDiskSize configurable over API

### LINKED ISSUE #3251: Make the root disk size of Openstack instances configurable
**User Story**

As a user I want to configure the size of the rootdisk of my Openstack instances

**Acceptance criteria**

* It is possible to configure the root disk size of Openstack instances via Kubermatic

### PR DESCRIPTION
**Pull request by bashofmann**
**What this PR does / why we need it**:
This adds support to the API to make the root disk size of OpenStack nodes configurable. Support was already added to machine-controller in https://github.com/kubermatic/machine-controller/pull/579.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #3251

**Does this PR introduce a user-facing change?**:

```release-note
Disk size of OpenStack nodes is now configurable
```

### CODE REVIEW

File: api/pkg/api/v1/types.go [hunk 1]
```diff
@@ -909,6 +909,9 @@ type OpenstackNodeSpec struct {
 	// Defines whether floating ip should be used
 	// required: false
 	UseFloatingIP bool `json:"useFloatingIP,omitempty"`
+	// if set, the rootDisk will be a volume. If not, the rootDisk will be on ephemeral storage and its size will be derived from the flavor
+	// required: false
+	RootDiskSizeGB int64 `json:"diskSize"`
```
Comment (alvaroaleman) [inline api/pkg/api/v1/types.go]: Wouldn't it be nicer to make this a `*int` so we can directly put it onto the machine without further checks?

Comment (bashofmann) [reply api/pkg/api/v1/types.go]: I made it the same type as the AWS disk size, but I guess from the json side it does not matter and it makes things easier in the code. I'll change it.

**Pull request merged**